Twig/Extension/CmfSeoExtension.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2015 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @author Ben Glassman <bglassman@gmail.com>
+ */
+
+namespace Symfony\Cmf\Bundle\SeoBundle\Twig\Extension;
+
+use Symfony\Cmf\Bundle\SeoBundle\SeoPresentationInterface;
+
+class CmfSeoExtension extends \Twig_Extension
+{
+    /**
+     * @var SeoPresentationInterface
+     */
+    protected $seoPresentation;
+
+    /**
+     * @param SeoPresentationInterface $seoPresentation
+     */
+    public function __construct(SeoPresentationInterface $seoPresentation)
+    {
+        $this->seoPresentation = $seoPresentation;
+    }
+
+    /**
+     * @{inheritdoc}
+     */
+    public function getFunctions()
+    {
+        return array(
+            new \Twig_SimpleFunction('cmf_seo_update_seo_page',
```

Comment:
sorry for not being responsive, i was away. i created #264 as its an interface change.

for the twig function, could we call it `cmf_seo_update_from_document`? the name you currently have looks weird with the repeated seo. and i rather have a good name here than exact naming consistency with something we agree is not well named.

@ElectricMaxxx wdyt?